### ISSUE TITLE
Ichimoku + RSI + stoploss 

### ISSUE THREAD

**Issue by onthetunnel** [freqtrade/freqtrade-strategies#18]
Hello , can someone implements this strategy? Thanks You !

**Comment by Dante2333** [comment]
I can implement it for you if you're willing to pay for it. If not, here are the instructions:

You can either use technical repo or FinTA, install those in pip3 on a linux server where you are running freqtrade.

From there, you'll be able to use the ichimoku strategy, rsi can be implemented from FinTA or Talib or qtpylib and the stop-loss is built in inside freqtrade. 

https://cryptocue.io/ichimoku-strategy-with-freqtrade/ this post contains how to implement an ichimoku strategy

Custom technical indicators can be installed through here: https://cryptocue.io/custom-technical-indicators-for-freqtrade/

FinTA https://cryptocue.io/finta-for-freqtrade/
An idea of how to use RSI can be deduced from here https://cryptocue.io/bb-and-rsi-strategy-freqtrade/

I'm willing to discuss more if you hit me up on telegram or the slack @dante2333 

I think you should close this @xmatthias

**Comment by xmatthias** [comment]
well, it's a valid request to ask for a new strategy implementation. 

If someone has an idea, but lacks the knowledge of implementing it, then this may help bring traders and developers together - so if i were to follow this suggestion to close this, then i should close all requests for new strategies.

the only thing this lacks to me some information on how to combine these indicators... 

@onthetunnel 
The information you provide above does not really point out how this should be used.

Ichimoku exists of 5 indicator lines - so you'll need to be a bit preciser on how to combine this with RSI. 
Simply throwing them together somehow and doing "some" kind of buy signal may not produce very good gains.

**Comment by berlinguyinca** [comment]
Sounds rather trivial to do, but do we want to utilize this as forum of
coding for hire? No disrespect meant in any possible way. but might be
better to have a dedicated consulting website for this.

**Comment by xmatthias** [comment]
who talks about coding for hire? 

if someone has an idea on how to implement a strategy, i think it's valid to request that so someone else can take a look at it and implement this, since it may inspire someone with the necessary skills.

I don't appreciate offering help for payment (for hire) here - requests filled here should be added as PR to this repository so the community can benefit.

@Dante2333 I've somehow missed this part - would appreciate if you could remove that part from your comment.

**Comment by Youbadawy** [comment]
I agree with @xmatthias ! I am certain there are a lot of traders that have concrete strategies that would be quite powerful to utilize, that us as developers might have less experience dealing with! Nonetheless a lot of developers have coding capabilities that might optimize the implementation of that said strategy, thus the cooperation of the two specializations will definitely add value ! 

Although let us not lose sight of those who seek to reap the value of open source without contributing.

Looking forward to assist in the enforcing, add value, and encourage the cooperation of both developers and traders! 

What would be nice is a discussion area where we could share ideas and discuss opportunities? is there a specific area like that for freqtrade?

**Comment by xmatthias** [comment]
I'd say that's either the Discord server or Slack.
It's unfortunate that at the moment, it's split across the 2 (however discord offers a ton of advantages, like unlimited history).
